### DISCUSSION TITLE
dbName is not working correctly

### DISCUSSION THREAD

**Discussion by mat-twg** [payloadcms/payload#9058] [Q&A]
### Describe the Bug

I need a table in **my** (not yours) database named something like `history` but I have `histories` and I'm using mongoDB

example collection
```
export const History: CollectionConfig = {
  slug: 'history',
  labels: {
    singular: 'History',
    plural: 'History',
  },
  dbName: 'history',
...
```
expected `tableName` with defined `dbName` property is **histroty** not **histories**

### Reproduction Steps

create new collection `History` like above with dbName: string

### Which area(s) are affected? (Select all that apply)

db-mongodb

### Environment Info

Payload: ^2.0.0
Nodejs: 20.14

**Comment by mat-twg** [comment]
sorry that is for v2...

**Comment by r1tsuu** [comment]
Collection names in MongoDB are automatically pluralized, regardless of `dbName`. However, you can disable that behaviour with passing `autoPluralization: false` to [the adapter options](https://payloadcms.com/docs/database/mongodb#options).

```ts
mongooseAdapter({
  url: process.env.MONGODB_URI,
  autoPluralization: false
})
```

**Comment by mat-twg** [comment]
Ok, so why `autoPluralization` is only global?

For example i already have db with 34 collections and naming with `plural` looks pretty.. e.g media -> media, type -> types
But now i need my custom name for one table(collection), and what? I should rebuild all my collections? :cactus: 

If i define `dbName` it should override all other configs, and save collection name as-is..

[option: collection](https://mongoosejs.com/docs/guide.html#collection)
Mongoose by default produces a collection name by passing the model name to the utils.toCollectionName method. This method pluralizes the name. Set this option if you need a different name for your collection.

**Reply by DanRibbens** [reply]
> If i define dbName it should override all other configs, and save collection name as-is.

I agree with this statement and implemented it here:
https://github.com/payloadcms/payload/pull/9107/files

This will be merged for v3.
